storage/innobase/mtr/mtr0mtr.cc

```diff
@@ -468,8 +468,10 @@ void mtr_t::commit()
       mysql_mutex_unlock(&buf_pool.flush_list_mutex);
     }
   }
-  else
-    m_memo.for_each_block_in_reverse(CIterate<ReleaseAll>());
+  else {
+    ReleaseAll rl;
+    m_memo.for_each_block_in_reverse(CIterate<ReleaseAll>(rl));
+  }
```

Comment:
Monty wants us to deviate from the K&R formatting style by adding a line break before each `{` (except in the `switch` statement).

Reply:
done